### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by mschmauder-hub**
#15 

Added a card component with text and an image component. Additionally, I  added a colors and fonts config file, so that I will be able to adjust styles and colors in one place. I also added a ScrollView, but I might change it to a Flatlist later.

Card component

### CODE REVIEW

File: SeriesMaster-Frontend/App/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+`;
+
+const Card = ({ children }) => {
+  return <CardContainer>{children}</CardContainer>;
+};
+
+export default Card;
+
+Card.propTypes = {
+  children: PropTypes.any,
```
Comment (lmachens) [inline SeriesMaster-Frontend/App/components/Card.js]:
```suggestion
  children: PropTypes.node,
```

File: SeriesMaster-Frontend/App/components/CardImage.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+`;
+
+const CardImage = ({ image }) => {
+  return <StyledCardImage source={image} />;
+};
+
+export default CardImage;
+
+CardImage.propTypes = {
+  image: PropTypes.any,
```
Comment (lmachens) [inline SeriesMaster-Frontend/App/components/CardImage.js]: I would change the name to clarify, that this is the path to the source. 

```suggestion
  imageSrc: PropTypes.string,
```

**Review by lmachens — APPROVED** [review]
Good job!

File: SeriesMaster-Frontend/App/Screens/DiscoverScreen.js [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
 const Main = styled.View`
-  align-items: center;
-  background: lightgrey;
+  background: ${colors.dark};
   width: 100%;
   flex: 1;
 `;
+const CardsContainer = styled.View`
+  flex-direction: row;
+  flex-wrap: wrap;
```
Comment (marwinburesch) [inline SeriesMaster-Frontend/App/Screens/DiscoverScreen.js:26]: you could use the `flex-flow` shorthand here :)
👐

File: SeriesMaster-Frontend/App/components/CardImage.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styled from "styled-components/native";
+import PropTypes from "prop-types";
+
+const StyledCardImage = styled.Image`
+  height: 125px;
+  width: 125px;
```
Comment (marwinburesch) [inline SeriesMaster-Frontend/App/components/CardImage.js:7]: These should be set by a grid or flexbox, could be in a separate PR no probs

**Review by marwinburesch — APPROVED** [review]
Nice and clean! Ooohwhee ✨
Excellent idea with the variables for colors
🐜

File: SeriesMaster-Frontend/App/components/CardImage.js (hunk 4 above)
Comment (mschmauder-hub) [reply SeriesMaster-Frontend/App/components/CardImage.js:7]: Yes, I will try to add it when I generate my Elements from the JSON server. Thanks!
